<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntityConditionDialog</class>
 <widget class="QDialog" name="EntityConditionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Entity type:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QComboBox" name="entityTypeCombo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabSpacing">
      <number>4</number>
     </property>
     <widget class="QWidget" name="general">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>662</width>
        <height>514</height>
       </rect>
      </property>
      <attribute name="label">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="NBTEdit"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Team:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Distance</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="labelAlignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <item row="0" column="1">
           <widget class="NumericInput" name="absDistanceInput">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="NumericInput" name="horizDistanceInput">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="NumericInput" name="xDistanceInput">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="NumericInput" name="yDistanceInput">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="NumericInput" name="zDistanceInput">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Absolute:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Horizontal</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>X:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Y:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Z:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>NBT:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="teamEdit"/>
       </item>
       <item row="3" column="2" rowspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>2</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Equipment</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <widget class="DialogDataButton" name="headPropBtn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Head...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="DialogDataButton" name="offhandPropBtn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Offhand...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="DialogDataButton" name="chestPropBtn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Chest...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="DialogDataButton" name="mainhandPropBtn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Mainhand...</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="DialogDataButton" name="legsPropBtn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Legsâ€¦</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="DialogDataButton" name="feetPropBtn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Feet...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Flags</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="TrueFalseBox" name="onFireCheck">
            <property name="text">
             <string>On fire</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="TrueFalseBox" name="isSneakingCheck">
            <property name="text">
             <string>Sneaking</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="TrueFalseBox" name="isSprintingCheck">
            <property name="text">
             <string>Sprinting</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="TrueFalseBox" name="isSwimmingCheck">
            <property name="text">
             <string>Swimming</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="TrueFalseBox" name="isBabyCheck">
            <property name="text">
             <string>Baby</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="DialogDataButton" name="locatPropBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Location...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="effects">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>662</width>
        <height>514</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Effects</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="4" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Duration:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="NumericInput" name="effectDuraInput">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QComboBox" name="effectCombo"/>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="effectAmbientCheck">
         <property name="text">
          <string>Ambient</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="NumericInput" name="effectAmpInput">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Amplifier:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="3">
        <widget class="QTableView" name="effectTableView">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>70</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Status effect:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3">
        <widget class="QPushButton" name="effectAddBtn">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="effectVisibleCheck">
         <property name="text">
          <string>Visible</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="player">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>662</width>
        <height>514</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Player</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Game mode:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="gameModeCombo">
           <item>
            <property name="text">
             <string>(not selected)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Survival</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Creative</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Adventure</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Spectator</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>Levels</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="NumericInput" name="playerLevelInput">
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>4</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Advancements</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>ID;</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLineEdit" name="advanmEdit"/>
          </item>
          <item row="1" column="0" rowspan="2" colspan="3">
           <widget class="QCheckBox" name="advanmCheck">
            <property name="text">
             <string>Granted</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="advanmAddBtn">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="3">
           <widget class="QTableView" name="advanmTableView">
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustToContents</enum>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Recipes</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_25">
            <property name="text">
             <string>ID:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLineEdit" name="recipeEdit"/>
          </item>
          <item row="1" column="0" rowspan="2" colspan="3">
           <widget class="QCheckBox" name="recipeCheck">
            <property name="text">
             <string>Granted</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="recipeAddBtn">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="3">
           <widget class="QTableView" name="recipeTableView">
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustToContents</enum>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>5</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Stats</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_26">
              <property name="text">
               <string>Type:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="statTypeCombo"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_27">
              <property name="text">
               <string>Stat:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="statEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_28">
              <property name="text">
               <string>Value:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="NumericInput" name="statValueInput">
              <property name="minimumSize">
               <size>
                <width>130</width>
                <height>22</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="QPushButton" name="statAddBtn">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableView" name="statsTableView">
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NumericInput</class>
   <extends>QFrame</extends>
   <header location="global">NumericInput.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DialogDataButton</class>
   <extends>QLabel</extends>
   <header location="global">dialogdatabutton.h</header>
  </customwidget>
  <customwidget>
   <class>TrueFalseBox</class>
   <extends>QCheckBox</extends>
   <header location="global">truefalsebox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EntityConditionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EntityConditionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
